Shoplifter/IGenericModConfigMenuApi.cs

```diff
@@ -0,0 +1,177 @@
+using System;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using StardewModdingAPI;
+using StardewModdingAPI.Utilities;
+using StardewValley;
+
+namespace GenericModConfigMenu
```

Comment:
This should be changed to use the Shoplifter namespace. This means a GenericModConfigMenu directive is not needed in the Modentry class.